MQ2AutoSize.cpp

```diff
@@ -350,6 +356,19 @@ void ChangeSize(PlayerClient* pChangeSpawn, float fNewSize)
 	}
 }
 
+
+void HandleResize(PlayerClient* pSpawn, bool bReset, int size, bool option) {
+	if (option) {
+		// only resize if functionality is enabled
+		if (AS_Config.Enabled) {
+			ChangeSize(pSpawn, bReset ? ZERO_SIZE : size);
+		}
+		else {
+			ChangeSize(pSpawn, ZERO_SIZE);
+		}
+	}
+}
```

Comment:
every place this is used, you have already checked `option` before calling, so I see no reason to pass this parameter.

```suggestion

void HandleResize(PlayerClient* pSpawn, bool bReset, int size, bool option)
{
	// only resize if functionality is enabled
	ChangeSize(pSpawn, !AS_Config.Enabled || bReset ? ZERO_SIZE : size);
}
```

Reply:
Refactored left overs. Thanks for catching it.